vignettes/readr.Rmd

```diff
@@ -127,7 +127,7 @@ parse_number("$1,234")
 
 There are two parsers that will never be guessed: `col_skip()` and `col_factor()`. You will always need to supply these explicitly.
 
-You can see the specification that readr would generate for a column file by using `spec_csv()`, `spec_tsv()` and so on:
+You can see the specification that readr would generate for a file by using `spec_csv()`, `spec_tsv()`, and so on.
 
 ```{r}
 x <- spec_csv(readr_example("challenge.csv"))
```

Comment:
Wouldn't it make sense to reveal the spec? I.e. why are we assigning the result to `x`?

(I have fetched the PR and rendered the vignette now. I assume you are also looking at the rendered result as you go.)

Reply:
This is how the example was when I started editing the file and I didn't see a reason to modify it since we aren't doing a total overhaul right now and the goal is to ensure that the `problems()` example has actual problems. Also, the behavior of `spec_*()` is currently in-flight https://github.com/tidyverse/readr/pull/1436 so some of the content here might change.